.circleci/config.yml

```diff
@@ -381,11 +379,11 @@ workflows:
           type: approval
           requires:
             - android-integration-test-build
-          <<: *release-branches-and-main
+          <<: *except-main-and-release
       - run-firebase-tests:
           requires:
             - hold
-          <<: *release-branches-and-main
+          <<: *except-main-and-release
```

Comment:
With this change, firebase tests will run:
- Never in main branch
- Always on release branches
- At least once per PR (if we want to have the green check before merging)